application/models/user_test.go

```diff
@@ -696,3 +696,74 @@ func (ms *ModelSuite) TestUser_AttachPhoto() {
 		ms.Fail("user.GetPhotoURL failed, %s", err)
 	}
 }
+
+func CreateUserFixturesForNicknames(ms *ModelSuite, t *testing.T) User {
+	ResetTables(t, ms.DB)
+
+	unique := domain.GetUuid().String()
```

Comment:
I like `t.Name()` because it's informative. Granted, it may not work if test interations > 1. Not sure about that.
```suggestion
	unique := t.Name()
```

Reply:
Done.